helpers/helpers.v1.d/go

```diff
@@ -90,7 +80,7 @@ ynh_use_go() {
     ynh_go_load_path="PATH=$PATH"
 
     # Sets the local application-specific Go version
-    pushd $install_dir
+    pushd ${install_dir:-$install_dir}
```

Comment:
What's the meaning of this substitution ?

Reply:
Bad copy paste from: https://github.com/YunoHost/yunohost/blob/a96cb9bb2c1a74806b9571daf4009652d46b35bb/helpers/helpers.v1.d/ruby#L95